docs/FAQ.md

```diff
@@ -6,15 +6,16 @@ You will need to setup several volume mounts as well as some environment
 variables:
 ```
 docker run --name telegraf
-	-v /:/hostfs:ro
 	-v /etc:/hostfs/etc:ro
 	-v /proc:/hostfs/proc:ro
 	-v /sys:/hostfs/sys:ro
-	-v /var/run/utmp:/var/run/utmp:ro
+	-v /var:/hostfs/var:ro
+	-v /run:/hostfs/run:ro
 	-e HOST_ETC=/hostfs/etc
 	-e HOST_PROC=/hostfs/proc
 	-e HOST_SYS=/hostfs/sys
-	-e HOST_MOUNT_PREFIX=/hostfs
```

Comment:
This is used by the disk input to strip the `/hostfs` from paths, are you sure we want to remove this?

Reply:
sorry, I make a mistake and will change it, thank you!